buildconfig/Jenkins/Conda/nightly_build_and_deploy.jenkinsfile

```diff
@@ -21,28 +22,35 @@ def conda_build_UNIX(){
 
 def conda_build_linux() {
     conda_build_UNIX()
-    archive:
-        includes: "mambaforge/envs/mantid-conda-build-env/conda-bld/linux-64/*.tar.bz2"
+    archive_conda_build("linux-64")
 }
 
 def conda_build_macos() {
     conda_build_UNIX()
-    archive:
-        includes: "mambaforge/envs/mantid-conda-build-env/conda-bld/osx-64/*.tar.bz2"
+    archive_conda_build("osx-64")
 }
 
 def conda_build_win(){
     bat "\"C:\\Program Files\\git\\bin\\bash.exe\" -c \"C:/Jenkins/workspace/$JOB_NAME/buildconfig/Jenkins/Conda/conda-build-recipes C:/Jenkins/workspace/$JOB_NAME --build-mantid --build-qt --build-workbench\""
-    archive:
-        includes: "mambaforge/envs/mantid-conda-build-env/conda-bld/win-64/*.tar.bz2"
+    archive_conda_build("win-64")
+}
+
+def archive_conda_build(build_name) {
```

Comment:
Good refactoring and good that this still works if the environment variable is not defined.